Testing/functional/e2e/cypress/integration/report/report.js

```diff
@@ -0,0 +1,32 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('Reports', () => {
+    before(() => {
+        cy.readConfig().as("config").then(config => {
+            cy.server();
+            cy.route('GET', '/v1/api/configuration/', config);
+            cy.login(Cypress.env('keycloak.username'), Cypress.env('keycloak.password'), AuthMethod.KeyCloak, "/reports");
+        })
+    })
+
+    it('Validate Service Selection', () => {       
+        cy.get('[data-testid=exportRecordBtn]')
+            .should('not.be.enabled', 'be.visible')
+
+        cy.get('[data-testid=reportType]')
+            .should('be.enabled', 'be.visible')
+            .select("MED")        
```

Comment:
Are we missing validations for COVID Result and MSP Visit options per "Have a selection module with services available and selectable (Medication, COVID Result, MSP Visit)" requirement on 9608 ?

Reply:
we don't have those options available yet and they should be covered by the next user stories